math/src/field/traits.rs

```diff
@@ -203,6 +203,9 @@ pub trait IsPrimeField: IsField {
     /// Returns an `CreationError::InvalidHexString`if the value is not a hexstring
     fn from_hex(hex_string: &str) -> Result<Self::BaseType, CreationError>;
 
+    #[cfg(feature = "std")]
+    fn to_hex(a: &Self::BaseType) -> String;
+
```

Comment:
It's missing the documentation on how this function should behave. It should have three bars /// and at least say what it does and the format of the output